website/docs/docs/build/snapshots.md

```diff
@@ -303,9 +473,20 @@ This strategy handles column additions and deletions better than the `check` str
 #### Ensure your unique key is really unique
 The unique key is used by dbt to match rows up, so it's extremely important to make sure this key is actually unique! If you're snapshotting a source, I'd recommend adding a uniqueness test to your source ([example](https://github.com/dbt-labs/jaffle_shop/blob/8e7c853c858018180bef1756ec93e193d9958c5b/models/staging/schema.yml#L26)).
 
+<VersionBlock lastVersion="1.8">
+
 #### Use a `target_schema` that is separate to your analytics schema
 Snapshots cannot be rebuilt. As such, it's a good idea to put snapshots in a separate schema so end users know they are special. From there, you may want to set different privileges on your snapshots compared to your models, and even run them as a different user (or role, depending on your warehouse) to make it very difficult to drop a snapshot unless you really want to.
 
+</VersionBlock>
+
+<VersionBlock firstVersion="1.9">
+
+#### Use a schema that is separate to your analytics schema
+Snapshots cannot be rebuilt. As such, it's a good idea to put snapshots in a separate schema so end users know they are special. From there, you may want to set different privileges on your snapshots compared to your models, and even run them as a different user (or role, depending on your warehouse) to make it very difficult to drop a snapshot unless you really want to.
```

Comment:
Based on what I've read, this is still the suggested practice - do we want to expand on this or is the text fine as-is?

Reply:
This is still best practice! Not sure about the "to your analytics schema" wording... maybe something like "Use a schema that is separate to your models' schema(s)"